website/docs/r/application_epg.html.markdown

```diff
@@ -64,7 +71,14 @@ resource "aci_application_epg" "fooapplication_epg" {
 
 * `relation_fv_rs_sec_inherited` - (Optional) Relation represents that the EPG is inheriting security configuration from other EPGs (Point to class fvEPg). Cardinality - N_TO_M. Type - Set of String.
 
-* `relation_fv_rs_node_att` - (Optional) Relation used to define a Static Leaf binding (Point to class fabricNode). Cardinality - N_TO_M. Type - Set of String.
+* `relation_fv_rs_node_att` - (Required) A block representing the relation to a Static Leaf binding (Point to class fabricNode). Cardinality - N_TO_M. Type: Block.
+
+  - `node_dn` - (Required) Distinguished name of the node. Type: String.
```

Comment:
Please change all the attribute descriptions to follow: The ... of the Node object. Type: String.

```suggestion
  - `node_dn` - (Required) The Distinguished Name of the Node object. Type: String.
```